sql/telemetry_derived/fxa_users_services_daily_v1/query.sql

```diff
@@ -0,0 +1,104 @@
+CREATE TEMP FUNCTION
```

Comment:
This query is basically identical to fxa_users_daily but windows by `user_id, service` rather than `user_id`.